tests/gold_tests/proxy_protocol/proxy_protocol.test.py

```diff
@@ -227,7 +225,7 @@ def setupTS(self, tr: 'TestRun') -> None:
         )
 
         self._ts.Disk.remap_config.AddLine(
-            f"map / http://backend.pp.origin.com:{self._server.Variables.external_port}/")
+            f"map / https://backend.pp.origin.com:{self._server.Variables.external_port}/")
```

Comment:
Why don't you add a new test? Is current test invalid?

Reply:
Good point. Added some https pp_out tests in addition to the existing http ones.